rewrite-xml/src/main/java/org/openrewrite/xml/RemoveContentVisitor.java

```diff
@@ -25,10 +25,17 @@ public class RemoveContentVisitor<P> extends XmlVisitor<P> {
 
     private final Content scope;
     private final boolean removeEmptyAncestors;
+    private final boolean removePrecedingComment;
 
+    @Deprecated
     public RemoveContentVisitor(Content tag, boolean removeEmptyAncestors) {
+        this(tag, removeEmptyAncestors, false);
```

Comment:
I'm leaning towards making the new behavior `true` by default, then inlining and removing the old constructor such that folks consciously have to choose to adopt the new default or not.
```suggestion
        this(tag, removeEmptyAncestors, true);
```